sentry-kotlin-multiplatform/src/appleMain/kotlin/io/sentry/kotlin/multiplatform/SentryApple.kt

```diff
@@ -6,12 +6,6 @@ import platform.Foundation.NSError
 import platform.Foundation.NSException
 
 internal actual object SentryBridge {
-    actual fun start(context: Any?, configuration: OptionsConfiguration<SentryKMPOptions>) {
```

Comment:
Does iOS accept `init` as an extension so it would be called `init` on both SDKs?

Reply:
It's possible to extend it with init but not use it in iOS, it throws a compile error if I try to call Sentry.init because it's a reserved keyword

Reply:
ok gotcha, lets keep `start` then for iOS.